waspc/src/Wasp/Generator/Crud/Routes.hs

```diff
@@ -0,0 +1,21 @@
+module Wasp.Generator.Crud.Routes where
+
+import Data.List (intercalate)
+import qualified Wasp.AppSpec.Crud as AS.Crud
+
+getRoute :: AS.Crud.CrudOperation -> String
+getRoute operation = case operation of
+  AS.Crud.Get -> "get"
+  AS.Crud.GetAll -> "getAll"
+  AS.Crud.Create -> "create"
+  AS.Crud.Update -> "update"
+  AS.Crud.Delete -> "delete"
```

Comment:
Following usual URL naming conventions, we should make the routes kebab-case (i.e., `get-all` instead of `getAll`).

Also, I think you can avoid the pattern match here by transforming the Enum into a string and calling `camelToKebabCase` (not sure).